module-utils/utility/Utils.hpp

```diff
@@ -23,6 +23,26 @@ namespace utils
     std::string bytesToHex(const std::vector<std::uint8_t> &bytes);
     std::vector<std::uint8_t> hexToBytes(const std::string &hex);
 
+    template <typename T> inline char singleDigitToChar(T digit)
+    {
+        static_assert(std::is_integral<T>::value, "Integral number required.");
```

Comment:
isn't digit singular by definition?

Reply:
it should be, but there's no type for it